github/client.go

```diff
@@ -371,6 +371,20 @@ func (client *Client) DeleteReleaseAsset(asset *ReleaseAsset) (err error) {
 	return
 }
 
+func (client *Client) DownloadReleaseAsset(url string) (asset io.ReadCloser, err error) {
+	api, err := client.simpleApi()
+	if err != nil {
+		return
+	}
+
+	asset, err = api.GetBody(url, "application/octet-stream")
```

Comment:
Could we avoid adding a new `GetBody` function and simply use `api.Get` to obtain a response object and return its `.Body` property from this function? I think it will result in less overall added code.